packages/legend-extension-external-store-service/src/models/metamodels/pure/model/packageableElements/store/serviceStore/mapping/ServiceMapping.ts

```diff
@@ -30,6 +31,7 @@ export class ServiceMapping implements Hashable {
   owner!: RootServiceInstanceSetImplementation;
   service!: ServiceStoreService;
   parameterMappings: ServiceParameterMapping[] = [];
+  path: object | undefined;
```

Comment:
Lets add this note for this `// @MARKER GENERATED MODEL DISCREPANCY --- Studio does not process lambda`. See other places we have done this for (lambda)

Reply:
done